requirements.txt

```
@@ -3,4 +3,4 @@ pytest-html
 pytest-cov
 flake8
 antlr4-python3-runtime
-edk2-pytool-library==0.10.13
\ No newline at end of file
+edk2-pytool-library>=0.10.13
```

edk2-pytool-library does not use major versions and updates the minor version seldomly when there is a breaking change.

I recommend setting the version to the latest release [v0.21.9](https://github.com/tianocore/edk2-pytool-library/releases/tag/v0.21.9)

And use the `~=` operator to match patch versions, which typically run a while. For example, the current patch version is `9`. Then you can deal with any larger changes (minor or major) when they're brought to your attention through a mechanism like dependabot at which time you can evaluate if any integration changes are required and PR feedback of the incoming change.

```suggestion
edk2-pytool-library~=0.21.9
```

---

Thanks!